Webpack/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./src/index.js",
+  "dependencies": {
+    "webpack-cli": "^4.9.1",
+    "webpack": "^5.65.0",
+    "webpack-dev-server": "^4.7.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.5",
+    "@babel/preset-env": "^7.16.5",
+    "@popperjs/core": "^2.11.0",
+    "babel-loader": "^8.2.3",
+    "bootstrap": "^5.1.3",
+    "clean-webpack-plugin": "^4.0.0",
+    "css-loader": "^6.5.1",
+    "css-minimizer-webpack-plugin": "^3.3.1",
+    "file-loader": "^6.2.0",
+    "html-loader": "^3.0.1",
+    "html-webpack-plugin": "^5.5.0",
+    "jquery": "^3.6.0",
+    "mini-css-extract-plugin": "^2.4.5",
+    "popper": "^1.0.1",
+    "sass": "^1.45.1",
+    "sass-loader": "^12.4.0",
+    "style-loader": "^3.3.1",
+    "webpack-merge": "^5.8.0"
+  },
+  "scripts": {
+    "dev": "webpack --config webpack.dev.js",
```

Я бы переименовала это, ведь этот скрипт запускает сборку, только для dev
```suggestion
    "build:dev": "webpack --config webpack.dev.js",
```